pyproject.toml

```diff
@@ -11,7 +11,7 @@ repository = "https://github.com/dusktreader/flask-praetorian"
 
 
 [tool.poetry.dependencies]
-python = "^3.6.2"
+python = "^3.7"
```

Comment:
The github actions should be modified to reflect this. We will also need to mention it in the docs. 

Also, please summarize these changes in the CHANGELOG under "unreleased".